egs/reverb/asr1/run.sh

```diff
@@ -330,12 +353,37 @@ if [ ${stage} -le 5 ]; then
     echo "Report the result"
     decode_part_dir=beam${beam_size}_e${recog_model}_p${penalty}_len${minlenratio}-${maxlenratio}_ctcw${ctc_weight}
     if [ $use_wordlm = true ]; then
-	decode_part_dir=${decode_part_dir}_wordrnnlm${lm_weight}
+	decode_part_dir=${decode_part_dir}_wordrnnlm${lm_weight}_${lmtag}
     else
-	decode_part_dir=${decode_part_dir}_rnnlm${lm_weight}
+	decode_part_dir=${decode_part_dir}_rnnlm${lm_weight}_${lmtag}
     fi
+    echo "RESULTS - Cln"
```

Comment:
Please make this result reporting part as a bash script and call it. Do not make `run.sh` very different from others

Reply:
Moved this to a file "local/get_results.sh"